bot/exts/filtering/_filters/domain.py

```diff
@@ -40,7 +40,7 @@ async def triggered_on(self, ctx: FilterContext) -> bool:
 
         for found_url in ctx.content:
             extract = tldextract.extract(found_url)
-            if self.content in found_url and extract.registered_domain == domain:
+            if self.content.lower() in found_url and extract.registered_domain == domain:
```

Comment:
Should `domain`  be lowercased too, since it's derived from `self.content`?

Reply:
Yeah you're right 743f2ce